src/main/scala/wordbots/CodeGenerator.scala

```diff
@@ -109,6 +111,11 @@ object CodeGenerator {
       case TheyP => "targets['theyP']()"
       case SavedTargetObject => "load('target')"
 
+      // Target tiles
+      case ChooseT(collection) => s"targets['choose'](${g(collection)})"
```

Comment:
This seems basically identical to the `ChooseO`, `AllC`, etc counterparts?

Reply:
Yeah, it is. Maybe I could combine the case statements.

Reply:
Ideally I would just have a single `All`, `Choose` and `Random` that were parametrized over the type of collection. But when I tried doing that, the parser couldn't handle the parametrized types.